texts/epn_paper.md

```diff
@@ -32,7 +32,10 @@ After several discussions we committed to developing a community platform for on
 <!-- #### In thinking about online events we had to revisit many assumptions about conferences. -->
 
 Once we started brainstorming online events, we realized, that despite those are similar to regular scientific meetings, a lot of our assumptions about conferences do not hold anymore.
-For example, the interactions between the speaker and the audience as well as among the audience themselves in an online event are quite different and much less spontaneous.
+For example, the interactions between the speaker and the audience as well as among the audience themselves in an online event are quite different and much less spontaneous. 
+As a speaker, for example, gauging the audience's reaction is much harder due to the lack of visual and audio contact -- most participants will have their audio muted and their video turned off. 
+This in turn makes setting the atmosphere more difficult, not knowing whether potential jokes were a success or not.
+To alleviate some of these challenges and to act as a catalyst between the audience and the speaker, one or multiple moderators are designated to keep an eye out for questions and relay those to the speaker.
```

Comment:
It is not clear to me how the moderators help the speaker with the jokes. Perhaps make a more direct connection.

Suggestion:
(...) one or multiple moderators are designated to keep an eye out for questions and reactions and transmit them to the speaker.

Reply:
```suggestion
This in turn makes setting the atmosphere more difficult, not knowing whether potential jokes were a success or not.
To alleviate some of these challenges and to act as a catalyst between the audience and the speaker, one or multiple moderators are designated to keep an eye out for questions and comments and relay those to the speaker.
```